index.html

```diff
@@ -5,6 +5,7 @@
 
 <h1 class="page-heading">EIPs
   <a href="https://gitter.im/ethereum/EIPs?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Gitter"></a>
```

Comment:
Should remove this tbh.

Reply:
Should be in a separate PR tho.

Reply:
Agree with @Pandapip1 that it might be worthy a separate PR